modules/formulize/templates/admin/form_permissions.html

```diff
@@ -21,7 +21,7 @@
                     <{/foreach}>
                 </select>
 								<br /><br />
-								<input type="button" name="showperms" value="Show permissions for these groups" />
+								<input type="button" name="showperms" value="<{$smarty.const._AM_PERMISSIONS_SHOW_PERMS_FOR_GROUPS}>" />

```

Comment:
I searched all the files for the _AM_PERMISSIONS_SHOW_PERMS_FOR_GROUPS constant, but it might only exist in the multiple permissions branch.